build/Makefile.glog

```diff
@@ -1,6 +1,6 @@
 UNAME := $(shell uname)
 ifeq ($(UNAME),FreeBSD)
-EXTRA_LDFLAGS="-lexecinfo"
+EXTRA_LDFLAGS="-lexecinfo -lunwind"
```

Comment:
Yes and no. This PR exposed an issue with the static build of glog. I tried this as a fix but as it didn't work I'm rolling it back.